### PR TITLE
Xc blockchain ubuntu

### PR DESCRIPTION
**Pull request by atcsecure**
--------------------MESSAGE FROM ADMIN, DELETE BEFORE SUBMITTING----------------------

Thank you for deciding to contribute to this repository, we're thrilled to have you contribute a template or a fix! :tada: Before we proceed, please be sure to review the contribution guidelines that will make the review process go smoother.
https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md

We've created an outline of recommended sections to fill out that will help make this Pull Request awesome!

--------------------MESSAGE FROM ADMIN, DELETE BEFORE SUBMITTING----------------------
### Changelog

*
*
*
*
*

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@atcsecure__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by azurecla** [comment]
__@atcsecure__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, AZPRBOT;

File: xc-blockchain-ubuntu/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+    },
+    "dnsLabelPrefix": {
+       "type": "string",
+       "metadata": {
+          "description": "Hostname assigned to the Public IP, also used as the VM Name. Must be lowercase. It should match with the following regular expression: ^[a-z][a-z0-9-]{1,61}[a-z0-9]$ or it will raise an error."
+       }
+    },
+    "installMethod": {
+      "type": "string",
+      "defaultValue": "From_Source",
```
Comment (singhkay) [inline xc-blockchain-ubuntu/azuredeploy.json]: Is there a binary install method available? Building from source takes a long time causing the user to pay for the build time on the VM

Comment (atcsecure) [inline xc-blockchain-ubuntu/azuredeploy.json]: At this time there is not, but I can put that together in a few days

**Pull request closed without merging**